docs/remote/troubleshooting.md

```diff
@@ -136,6 +154,19 @@ If you are running into problems with VS Code hanging while trying to connect (a
 
 Enable the `remote.SSH.showLoginTerminal` [setting](/docs/getstarted/settings.md) in VS Code and retry. If you are prompted to input a password or token, see [Enabling alternate SSH authentication methods](#enabling-alternate-ssh-authentication-methods) for details on reducing the frequency of prompts.
 
+**Work around a bug with Windows OpenSSH server 8.x**
```

Comment:
The bug actually starts with 7.9

Reply:
Interesting - Updated it to be more generic in terms of the specific versions.